### ISSUE TITLE
Make it clearer where actions can be passed & used

### ISSUE THREAD

**Issue by jenweber** [emberjs/guides#2306]
[This page](https://guides.emberjs.com/v3.0.0/templates/actions/) in the guides talks about actions, but doesn't cover some major gotchas around where actions should be put and where they can be passed.

Examples: 
- An action defined in a controller _can_ be passed to a route's child components
- An action in the route's js _can't_ be passed to a child component
- An action in a route js can be used in the route's template directly

This should be explained somehow.

**Comment by sbatson5** [comment]
Hi @jenweber 👋 .  I wouldn't mind taking a stab at this.  There is a lot to unpack when it comes to actions, however.  Would it make sense to split some of this up? For instance, perhaps moving some controller specific pieces to the controllers section?  There should also be some room for explaining what the action helper is doing for us (i.e. setting the context of the function).  It just feels like it's a bit bigger than just the `Templates` section 🤔

**Comment by jenweber** [comment]
@sbatson I agree that it’s a bigger issue than one section. I recommend separate PRs so that changes are easier to review. But you can drop some ideas here in this issue for brainstorming
